src/index.ts

```diff
@@ -1,61 +1,171 @@
 import axios from 'axios';
 
-type SuggestResult = [
+type GoogleSuggestResult = [
```

Comment:
maybe we should move each provider result type and class into its own file under `src/providers/`, export them from there and import it in here, so each provider can be maintained separately and `index.ts` doesn't grow too much in size if we add more providers in the future?